src/dotnet-scaffolding/Microsoft.DotNet.Scaffolding.Helpers/Steps/AddPackageStep/AddPackagesStep.cs

```diff
@@ -0,0 +1,31 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+using Microsoft.DotNet.Scaffolding.Helpers.General;
+using Microsoft.DotNet.Scaffolding.Helpers.Services;
+
+namespace Microsoft.DotNet.Scaffolding.Helpers.Steps.AddPackageStep;
+
+internal class AddPackagesStep : ScaffoldStep
+{
+    public required IList<string?> PackageNames { get; init; }
+    public required string ProjectPath { get; init; }
+    public required ILogger Logger { get; init; }
+    public bool Prerelease { get; set; } = false;
```

Comment:
Not for this PR, but just as a note that we probably need to refactor this generally to handle versions better (in other words, allow specifying a version in addition to just "latest" or "latest prerelease")

Reply:
sounds good yep